### PR TITLE
Upgrade to 514.1552

### PR DESCRIPTION
**Pull request by celotajstg**
## About The Pull Request

This pull request changes the minimum major version from 513 to 514. From what I can see it works decently enough to warrant switching or at the very least test-merging the switch.

I noticed quite a bit of complaining in #coderbus about when we will finally switch to 514, so there's that too.

## Why It's Good For The Game

Bug fixes and improvements that come from version 514.

## Changelog
:cl:
code: The minimum BYOND version is now 514.1552
/:cl:

### CODE REVIEW

**Comment by celotajstg** [comment]
Unsure why the linters break, but should be okay probably.

There is one known issue about ?.var++, however from what I can see it is not affecting our code:
http://www.byond.com/forum/post/2669054

image

**Comment by TiviPlus** [comment]
Bad idea, new 514 versions are evidently still broken, and this will also force everyone who codes to update for no reason

**Comment by TiviPlus** [comment]
Also you didnt update CI version

**Comment by Cyberboss** [comment]
Pretty sure the last live test of 514 didn't go so well, though I forget which version it was

**Comment by celotajstg** [comment]
> Also you didnt update CI version or fully remove extools

Where should the version be updated?



> Pretty sure the last live test of 514 didn't go so well, though I forget which version it was

When was the last live test, and can we schedule one on Terry this weekend?

**Comment by TiviPlus** [comment]
It was on 1549, which was also a stabler version, and you need a new spacemandmm release + spacemandmm version update for CI

**Comment by celotajstg** [comment]
Not sure if I get what I have to change, to be honest.

**Comment by Mothblocks** [comment]
Yeah, we still need to perform a lot more testing on this ourselves. A PR like this isn't gonna happen until *we're* on 514, otherwise we just break our own compilation.

File: code/game/objects/items/RCD.dm [hunk 1]
```diff
@@ -287,9 +287,7 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 		#warn Please replace the loop below this warning with an `as anything` loop.
 		#endif
 
-		for (var/_content_of_turf in surrounding_turf.contents)
-			// `as anything` doesn't play well on 513 with special lists such as contents.
-			// When the minimum version is raised to 514, upgrade this to `as anything`.
+		for (var/_content_of_turf as anything in surrounding_turf.contents)
```
Comment (Mothblocks) [inline code/game/objects/items/RCD.dm:290]: This isn't what that comment means, it means to remove `var/atom/content_of_turf` and make this `for (var/atom/content_of_turf as anything in surrounding_turf.contents)`

**Comment by LemonInTheDark** [comment]
To be a bit more clear, if we want to test 514 on master we can just have the keyholders update the server's byond version, that's supported by the code.

The only point at which we'd want to update the codebase to 514 is when we start to use stuff that only works for it, or that extools doesn't support. that day will come, but we're not there yet

**Pull request closed without merging**